src/main/resources/static/css/home-page-style.css

```diff
@@ -0,0 +1,81 @@
+body{
+    background-color: #42BFDD;
+
+}
+
+p{
+    font-size: 24px;
+  }
+
+h2{
```

Comment:
For consistent design, shouldn't these elements be styled globally for the whole site? I imagine an `h2` styling should be the same everywhere, for example